src/_projects/installfest-2023-1.md

```diff
@@ -0,0 +1,55 @@
+| Manjaro KDE 22.0.5.230316      | [manjaro-kde-22.0.5-230316-linux61.iso](https://download.manjaro.org/kde/22.0.5/manjaro-kde-22.0.5-230316-linux61.iso)                                                             | 3.8GB       | `7dc1b3330c7eb17ea97a97b5b7b943fba37027b340cd625f5cae7a6584342883` (SHA256) |
+| NixOS 22.11.3366 GNOME         | [nixos-gnome-22.11.3366.5b7cd5c39be-x86_64-linux.iso](https://releases.nixos.org/nixos/22.11/nixos-22.11.3366.5b7cd5c39be/nixos-gnome-22.11.3366.5b7cd5c39be-x86_64-linux.iso)     | 2.2GB       | `b07736a32301cdaa53191d6d09b10bb77fb2c17eaafd03d5accfd3c44f2ee5f6` (SHA256) |
+| NixOS 22.11.3366 Minimal       | [nixos-minimal-22.11.3366.5b7cd5c39be-x86_64-linux.iso](https://releases.nixos.org/nixos/22.11/nixos-22.11.3366.5b7cd5c39be/nixos-minimal-22.11.3366.5b7cd5c39be-x86_64-linux.iso) | 837MB       | `032e04ef6b3f8716eb1fa689dc746473b979ac7b41d392399e313e2e21fb8df9` (SHA256) |
+| Pop_OS 22.04 25                | [pop-os_22.04_amd64_intel_25.iso](https://iso.pop-os.org/22.04/amd64/intel/25/pop-os_22.04_amd64_intel_25.iso)                                                                     | 2.5GB       | `945baf9da2b9883d1cfc841a63e5ac3b52df1b9038beef8ef3c95142cede0346` (SHA256) |
+| Ubuntu 22.04.2 LTS             | [ubuntu-22.04.2-desktop-amd64.iso](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso)                                                                           | 4.6GB       | `b98dac940a82b110e6265ca78d1320f1f7103861e922aa1a54e4202686e9bbd3` (SHA256) |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|-----------------------------------------------------------------------------|
+
+## Voluntários
+
+|----------|-----------------|----------------------------------------------------|
+| **Nome** | **Email**       | **Horários**                                       |
+|----------|-----------------|----------------------------------------------------|
+| Gabriel  | g.fontes@usp.br | 12-13h, 16-19h. _Intermitente em outros horários._ |
+|----------|-----------------|----------------------------------------------------|
+
+## Pendrives
+
+|-------------|---------------------------------|----------|----------|
+| **Tamanho** | **Descrição**                   | **Dono** | **ISOs** |
+|-------------|---------------------------------|----------|----------|
+| 256GB       | Sandisk Prateado (c/ USB A e C) | Gabriel  | Todas    |
+| 128GB       | Sandisk Prateado                | Gabriel  | Todas    |
+|-------------|---------------------------------|----------|----------|
+
+## Guestbook
+
+|----------|-----------|-----------|--------------|----------------|
+| **Nome** | **Curso** | **Email** | **Instalou** | **Instalador** |
```

Comment:
Fico um pouco reticente sobre manter o email das pessoas na nossa página. Se for o institucional, tudo bem já que o ICMC já faz isso, mas o pessoal pode implicar certos problemas (leia-se LGPD e próprias questões de privacidade. Aguardo mais opiniões, mas minha sugestão é manter essas infos no nextcloud e depois publicar somente as estatísticas.

Reply:
Hmmm bom ponto. Vamos colocar aqui só se for institucional então. Se não for, a gente anota lá e coloca uma versão censurada aqui ou omite.